src/datachain/lib/dc.py

```diff
@@ -1934,6 +1934,7 @@ def from_csv(
         session: Optional[Session] = None,
         settings: Optional[dict] = None,
         column_types: Optional[dict[str, "Union[str, ArrowDataType]"]] = None,
+        newlines_in_values: bool = False,
```

Comment:
a suggestion: should we introduce a "parse_options` dict as a parameter that we can just pass into the parser then? (I'm a bit concerned about adding each parameter one by one here). WDYT?